src/lib.rs

```diff
@@ -306,6 +306,13 @@ extern crate unicode_xid;
 #[cfg(feature = "printing")]
 extern crate quote;
 
+#[cfg(doctest)]
```

Comment:
Note that this increases MSRV to 1.40. https://github.com/BurntSushi/termcolor/issues/35

Reply:
Outch, good point!